independent-projects/tools/analytics-common/src/main/java/io/quarkus/analytics/AnalyticsService.java

```diff
@@ -129,6 +131,14 @@ public void close() {
                 log.debug("[Quarkus build analytics] Build analytics sent successfully. Sent event can be seen at .../target/" +
                         fileLocations.lastTrackFileName());
             }
+        } catch (ExecutionException | TimeoutException e) {
+            if (log.isDebugEnabled()) {
+                log.debug("[Quarkus build analytics] Failed to send build analytics to Segment: " +
+                        e.getClass().getName() + ": " +
+                        (e.getMessage() == null ? "(no message)" : e.getMessage()));
+            }
+            log.info("[Quarkus build analytics] Failed to send build analytics to Segment. " +
```

Comment:
Why info?

Reply:
A couple of weeks ago ppl mentioned that WARN would scare ppl without a reason.

Reply:
I guess it was useful while we were testing it but now it is just be an annoyance. 
Will remove that and the remote config download message.

Reply:
fine to have it as debug. but yes; not shown by default as it confuses more than helping.